lib/rust/ensogl/example/complex-shape-system/src/lib.rs

```diff
@@ -95,15 +93,17 @@ pub fn main() {
     view1.size.set(Vector2::new(300.0, 300.0));
     view1.mod_position(|t| *t = Vector3::new(50.0, 50.0, 0.0));
 
-    let mask_layer = scene::layer::Layer::new(logger.sub("MaskLayer"));
+    let mask_layer = scene::layer::Layer::new("MaskLayer");
     scene.layers.node_searcher.set_mask(&mask_layer);
 
     let mask = mask::View::new();
     mask.size.set(Vector2::new(300.0, 300.0));
     mask.mod_position(|t| *t = Vector3::new(-50.0, 0.0, 0.0));
 
+    // FIXME: scissor box should not be computed from the left screen border. It should be affected
+    //     by the camera position.
```

Comment:
FIXME without owner. (Maybe should just be fixed)

Reply:
Thank you!